assignments/callbacks.js

```diff
@@ -27,25 +27,45 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
+getLength(items, function(len) {
+  console.log(len)});
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr[arr.length - 1]);
 }
+last(items, function(las) {
+  console.log(las)});
 
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x, y);
 }
+sumNums(1,2, function(x,y) {
+  console.log(x+y)});
 
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x, y);
 }
+sumNums(3,2, function(x,y) {
+  console.log(x*y)});
 
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+  if(list.includes(item)){
```

Comment:
Nice!  You used the `includes` method.  However, it does return a boolean, so a slicker solution could have been
```
function contains(item, list, cb) {
   return cb(list.includes(item)
}
```